src/site/stages/build/drupal/graphql/GetAllPages.graphql.js

```diff
@@ -18,6 +19,7 @@ module.exports = `
         ... page
       }
     }
+    ${sidebarQuery}
```

Comment:
I think we need to add this to `GetLatestPageById`, too?

Reply:
Agreed, @jbalboni. Maybe in GetPageById, too? (I don't know if we're using that one for anything.)

Reply:
I don't think we're using that one at the moment, but it wouldn't hurt to add it there, too.

Reply:
Just added the sidebar query to GetLatestPageById and GetPageById